INSTALL

```diff
@@ -257,7 +257,8 @@
 
   no-asm
                    Do not use assembler code. On some platforms a small amount
-                   of assembler code may still be used.
+                   of assembler code may still be used. This should be viewed
+                   rather as debugging/trouble-shooting option than production.
```

Comment:
This would be better, I think:

```
                   as debugging/trouble-shooting option rather than production.
```

Reply:
I also exchanged sentences. Because one could as well read it that "small amount of assembler code [on some platforms" is debugging/trouble-shooting thing...